modules/distr-tracing-tempo-config-query-frontend.adoc

```diff
@@ -8,60 +8,139 @@
 
 Query is a service that retrieves traces from storage and hosts the user interface to display them.
 
-.Parameters used by the {TempoOperator} to define Query
+.Configuration parameters for the Querier
 [options="header"]
-[cols="l, a, a, a"]
 |===
 |Parameter |Description |Values |Default value
 
-|spec:
-  query:
-    replicas:
-|Specifies the number of Query replicas to create.
-|Positive integer
+|`nodeSelector`
+|The simple form of the node-selection constraint.
+|type: object
+|
+
+|`replicas`
+|Specifies the number of replicas to be created for the component.
+|type: integer; format: int32
+|
+
+|`tolerations`
+|Specifies component-specific pod tolerations.
+|type: array
 |
 |===
 
 
-.Configuration parameters passed to Query
+.Configuration parameters for the Query Frontend
 [options="header"]
-[cols="l, a, a, a"]
 |===
 |Parameter |Description |Values |Default value
 
-|spec:
-  query:
-    options: {}
-|Configuration options that define the Query service.
+|`component`
+|Configuration of the Query Frontend component.
+|type: object
+|
+
+|`nodeSelector` (inside component)
+|The simple form of the node selection constraint.
+|type: object
+|
+
+|`replicas` (inside component)
```

Comment:
In this table too, I would suggest removing "Specifies" from this line and all other lines.

Reply:
Thank you, both, for the feedback. I'll remove `specifies`, sounds good to me.